architecture/additional_concepts/storage.adoc

```diff
@@ -123,7 +123,7 @@ to policy.
 === Reclaim volumes
 
 The reclaim policy of a `PersistentVolume` tells the cluster what to do with
-the volume after it is released. Volumes reclaim policy can either be `Retain` or `Delete`.
+the volume after it is released. A PV's reclaim policy can be either `Retain` or `Delete`.
 
 `Retain` reclaim policy allows manual reclamation of the resource for those volume plug-ins that support it. `Delete` reclaim policy deletes both the `PersistentVolume` object from {product-title} and the associated storage asset in external infrastructure, such as AWS EBS, GCE PD, or Cinder volume.
 
```

Comment:
I know this is text was already here. I wonder if it would be easier to see the two options if they were bullets like in the 4.x docs?
* `Retain` reclaim policy allows
* `Delete` reclaim policy deletes

Reply:
I agree, the bullets improve readability. Resolved.